### ISSUE TITLE
no byte?

### ISSUE THREAD

**Issue by InfiniteRegression** [libmir/mir-algorithm#100]
```d
	auto matrix = slice!byte(3, 4);
	matrix[] = 0;
	matrix.diagonal[] = 1;

	auto row = matrix[2];
	row[3] = 6;
```
fails but changing byte to double works fine?!?! Same issue with short, etc.

**Comment by 9il** [comment]
Try this for now:
```d
auto matrix = slice!byte(3, 4);
matrix[] = byte(0);
matrix.diagonal[] = byte(1);

auto row = matrix[2];
row[3] = byte(6);
```
0, 1, 6 is int (32 bits), so it does not understand that literals. Will try to fix it anyway.

**Comment by 9il** [comment]
This can not be fixed with current D features. Please use `byte(1)` or `cast(byte) 1` instead.

**Comment by 9il** [comment]
Fixed in v3.10.88
